.github/workflows/verify-build.yml

```diff
@@ -17,9 +17,13 @@ jobs:
         sudo apt -y install gcc-arm-none-eabi
     - name: Initialize submodules
       run: git submodule update --init --recursive
-#    - name: make simulator
-#      run: make simulator
-    - name: clean
-      run: make clean
     - name: make stm32f413
       run: make stm32f413
+    - name: clean
+      run: make clean
+    - name: make simulator
+      run: make simulator
+    - name: build CANbus
+      run: rm ./bps-simulator.out && make simulator TEST=CI_CANbus && mkdir BSP/Simulator/DataGeneration/Data
+    - name: run CANbus
+      run: sudo ./Continuous_Integration/CANbus/Test_CANbus.sh
```

Comment:
Should we add documentation to this PR on how to add this here (maybe in the Readme)?